src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/integration_tests/tests/web/VersionPageTests.kt

```diff
@@ -103,6 +111,10 @@ class VersionPageTests : IntegrationTest()
         val report = data[ORDERLYWEB_REPORT_VERSION_FULL.REPORT]
         val version = data[ORDERLYWEB_REPORT_VERSION_FULL.ID]
 
-        return Pair(report, "/report/$report/$version")
+        if (latest == true) {
+            return Pair(report, "/report/$report/latest/")
+        } else {
+            return Pair(report, "/report/$report/$version")
+        }
```

Comment:
A bit more idiomatic:
```suggestion
        return Pair(report, if (latest) "/report/$report/latest/" else "/report/$report/$version")
```